app/model/testdata/CreateEventData.scala

```diff
@@ -50,8 +50,8 @@ object CreateEventData {
       val id = createRequest.id.getOrElse(Random.Event.id)
       val eventType = createRequest.eventType.getOrElse(Random.Event.eventType)
       val description = createRequest.description.getOrElse(Random.Event.description)
-      val location = createRequest.location.getOrElse(Random.Event.location)
-      val venue = createRequest.venue.getOrElse(Random.Event.venue)
+      val location = createRequest.location.map(l => Location(l)).getOrElse(Random.Event.location)
```

Comment:
I think you can also change CreateEventRequest to use Venue and Location classes instead of Strings. Not necessary to do it now if you need to merge this soon.